modules/about-crio.adoc

```diff
@@ -6,6 +6,6 @@
 [id="about-crio_{context}"]
 = About CRI-O container runtime engine
 
-CRI-O is a Kubernetes-native container runtime implementation that integrates closely with the operating system to deliver an efficient and optimized Kubernetes experience. CRI-O provides facilities for running, stopping, and restarting containers.
+include::snippets/about-crio-snippet.adoc[]
 
-The CRI-O container runtime engine is managed using a systemd service on each {product-title} cluster node. When container runtime issues occur, verify the status of the `crio` systemd service on each node. Gather CRI-O journald unit logs from nodes that manifest container runtime issues.
+When container runtime issues occur, verify the status of the `crio` systemd service on each node. Gather CRI-O journald unit logs from nodes that manifest container runtime issues.
```

Comment:
Often we use _manifest_ to refer to YAML files in Kubernetes; I wonder if a different word might suit? Not critical, just a thought.

Reply:
@jboxman-rh Good catch. I didn't look at that line as it was existing text. Manifest, while technically correct, is the wrong word here.